schema/src/test/test_cwms_level.sql

```diff
@@ -387,7 +389,54 @@ begin
 
    ut.expect(l_count).to_equal(0);
 
+   ----------------------------------------------------------------------------
+   -- store a constant level and retrieve with retrieve_location_level2      --
+   -- https://github.com/HydrologicEngineeringCenter/cwms-database/issues/49 --
+   ----------------------------------------------------------------------------
+   cwms_level.store_location_level4(
+      p_location_level_id => c_top_of_normal_elev_id,
+      p_level_value       => 1000,
+      p_level_units       => c_elev_unit,
+      p_effective_date    => l_effective_date,
+      p_timezone_id       => c_timezone_id,
+      p_office_id         => c_office_id);
+   commit;
+
+   declare
+      l_value_out            number;
+      l_comment_out          varchar2(256);
+      l_effective_date_out   varchar2(64);
+      l_interval_origin_out  varchar2(64);
+      l_interval_months_out  integer;
+      l_interval_minutes_out integer;
+      l_interpolate_out      varchar2(1);
+      l_seasonal_values_out  varchar2(32767);
+   begin
+      cwms_level.retrieve_location_level2(
+         p_level_value       => l_value_out,
+         p_level_comment     => l_comment_out,
+         p_effective_date    => l_effective_date_out,
+         p_interval_origin   => l_interval_origin_out,
+         p_interval_months   => l_interval_months_out,
+         p_interval_minutes  => l_interval_minutes_out,
+         p_interpolate       => l_interpolate_out,
+         p_seasonal_values   => l_seasonal_values_out,
+         p_location_level_id => c_top_of_normal_elev_id,
+         p_level_units       => c_elev_unit,
+         p_office_id         => c_office_id);
+
+      dbms_output.put_line('l_level_value       = ' || l_value_out);
```

Comment:
while just not throwing an exception is good, there should still be ut.expect statements here instead of just printing it out. It's always possible a future change will accidentally break something.